integration-tests/features/selfcheck.feature

```diff
@@ -3,8 +3,6 @@ Feature: Check the implementation of test steps
   @selfcheck
   Scenario: Check the updated stack analysis response
     Given data directory exists
-    When I download and parse outlier probability threshold value
```

Comment:
Once we update the values, we can have these scenarios check that frequency_count should be >=0.

Reply:
Well, @tisnik don't we need to update these scenarios as well?

Reply:
yeah, good catch.

Reply:
I know it's been removed. But what I was saying was, shouldn't we add new scenarios for `frequency_count`?

Reply:
we did not delete any check. The `When I download and parse outlier probability threshold value` was needed to download the threshold value for `outlier_probability`, but because `frequency_count` has no configurable threshold (AFAIK), it is not needed.

Reply:
Ahh, right. Let me have a final look.